dependencies/vaticle/repositories.bzl

```diff
@@ -20,6 +20,6 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def vaticle_dependencies():
```

Comment:
We've updated the deploy_npm build tool, as the previous version was not fully reliable in how it passed the auth token to the tool.